core/templates/pages/exploration-editor-page/settings-tab/settings-tab.component.spec.ts

```diff
@@ -254,34 +254,44 @@ describe('Settings Tab Component', () => {
       testSubscriptipns.unsubscribe();
     });
 
+    // eslint-disable-next-line oppia/no-test-blockers
```

Comment:
Why is this required?

Reply:
This line was left in the code by mistake after the debugging phase. I have now removed this.